eng/pipelines/templates/stages/archetype-js-release.yml

```diff
@@ -89,6 +96,12 @@ stages:
                           arguments: '-pathToArtifacts $(Package.Archive) -accessLevel "public" -tag $(Tag) -additionalTag "$(AdditionalTag)" -registry ${{parameters.Registry}} -npmToken $(azure-sdk-npm-token)'
                           pwsh: true
 
+                      - pwsh: |
+                          eng/scripts/tools/remove-npm-next-tag.ps1 -packageArtifact $pkgArtifact -workingDirectory $(System.DefaultWorkingDirectory) -npmToken $(azure-sdk-npm-token)
```

Comment:
Is there any reason to do this post publish? Could we combine the scripts/steps?

If we keep this step separate it might be better to call it something like "cleanup" or "normalize-npm-tags" so it is clearer that this might not do anything if the tags are already correct.

Reply:
I like the naming change to cleanup. Reason for executing this as post step is make it clear that this is not changing tags of newly publishing version. But we are removing next tag from previously published preview version if that version is no longer in GA. I didn't want this step to be done before GA version is published. So intention is to run this only if GA version is published. I will change the name to make it more clear.